### ISSUE TITLE
Support CSS `filter` style

### ISSUE THREAD

**Issue by claell** [Kozea/WeasyPrint#1972]
Doesn't seem to be supported as of now. I didn't find an issue about this or any other related documentation for this and WeasyPrint.

Since I'd find the feature helpful, I am opening this issue.

**Comment by liZe** [comment]
That would be nice to have! Some filters are already handled by the PDF specification and should be quite "easy" to support, once the whole CSS parsing/validation code is done.

If someone wants to try, I can definitely help!

Related to #1962.

**Comment by grewn0uille** [comment]
Hello!

As you may know, two weeks ago was [CourtBouillon 3-year anniversary](https://www.courtbouillon.org/blog/00046-three-years-of-courtbouillon) 🎉.

For this occasion, we prepared [a short survey](https://thym.courtbouillon.org/4) to have your opinion on this year’s features and to know what you’d like to see in the future!
Don’t hesitate to give a boost to this feature ✨️

 The survey is opened until November 19.

**Comment by claell** [comment]
Just came to this again, when trying to find a way to blur an area of an image. If there is a different way how to do this within WeasyPrint, I am all ears.

Also see https://stackoverflow.com/questions/78534282/applying-css-filter-property-to-pdf-generated-with-weasyprint

**Comment by liZe** [comment]
> Just came to this again, when trying to find a way to blur an area of an image.

Unfortunately, as far as I know, there’s no blur operation in the PDF format. Other filters may be easy to add because they exist in PDF.

**Comment by claell** [comment]
Alright, thanks for letting me know! I plan to use a workaround for now (blurring the area of the image using Pillow beforehand). Probably less flexible, as it needs more planning and considerations, but also not too bad for now.
